cranelift/codegen/src/isa/x64/lower.rs

```diff
@@ -89,32 +86,98 @@ fn matches_input_any<C: LowerCtx<I = Inst>>(
     })
 }
 
+fn generate_constant<C: LowerCtx<I = Inst>>(ctx: &mut C, ty: Type, c: u64) -> Reg {
```

Comment:
Please add a 1-liner comment saying roughly what this does.

Reply:
Done.